runelite-api/src/main/java/net/runelite/api/Constants.java

```diff
@@ -52,11 +52,31 @@
 	 */
 	public static final double GAME_FIXED_ASPECT_RATIO = (double) GAME_FIXED_WIDTH / (double) GAME_FIXED_HEIGHT;
 
+	/**
+	 * The width of the game viewport when running in fixed mode.
+	 */
+	public static final int GAME_FIXED_VIEWPORT_WIDTH = 512;
+
+	/**
+	 * The height of the game viewport when running in fixed mode.
+	 */
+	public static final int GAME_FIXED_VIEWPORT_HEIGHT = 334;
```

Comment:
I've prefixed these constants with `GAME_` to be consistent with the stuff already here for `GAME_FIXED_WIDTH` and such, but is the `CLIENT_` prefix be better for these instead? (such as in `CLIENT_DEFAULT_ZOOM`)